### ISSUE TITLE
Publication metadata link can be mouse clicked but not ENTER key triggered (not keyboard accessible, though screen readers can emulate the click event)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#2300]
<img width="812" alt="Screenshot 2024-05-21 at 17 53 39" src="https://github.com/edrlab/thorium-reader/assets/625201/7197a3e4-4b7e-496c-87cb-a6917b0ac9f3">

**Comment by danielweck** [comment]
Reminder: VoiceOver users - for example - can activate the hyperlink with CTRL+OPT+SPACE but ENTER on its own doesn't work, just as it fails for keyboard users who do not use screen readers.
